pkg/brew/ripgrep-bin.rb

```diff
@@ -2,8 +2,18 @@ class RipgrepBin < Formula
   version '0.7.1'
   desc "Search tool like grep and The Silver Searcher."
   homepage "https://github.com/BurntSushi/ripgrep"
-  url "https://github.com/BurntSushi/ripgrep/releases/download/#{version}/ripgrep-#{version}-x86_64-apple-darwin.tar.gz"
-  sha256 "ee670b0fba46323ee9a2d1c5b8bee46fa3e45778f6f105f2e8e9ee29e8bd0d45"
+
+  case RUBY_PLATFORM
+  when /darwin/
+      url "https://github.com/BurntSushi/ripgrep/releases/download/#{version}/ripgrep-#{version}-x86_64-apple-darwin.tar.gz"
+      sha256 "ee670b0fba46323ee9a2d1c5b8bee46fa3e45778f6f105f2e8e9ee29e8bd0d45"
+  when /x86_64-linux/
+      url "https://github.com/BurntSushi/ripgrep/releases/download/#{version}/ripgrep-#{version}-x86_64-unknown-linux-musl.tar.gz"
+      sha256 "ac595c2239b9a30e0e0744578afa6b73e32cdd8ae61d4f1c0ee5d6b55adbadcf"
+  when /i686-linux/
+      url "https://github.com/BurntSushi/ripgrep/releases/download/#{version}/ripgrep-#{version}-i686-unknown-linux-musl.tar.gz"
+      sha256 "6e64e7481fbe6019a873a078af13a4aa86ec57a6f94765fd8b57b40d8138560e"
```

Comment:
Could you please remove the `i686` option? I don't want to devote resources to something people probably won't use. I realize CI produces builds for it, but only because it doesn't cause any trouble. However, if I support this on linuxbrew and decide one day to remove support for it, people might get upset.

Reply:
done.